benchmarks/table.py

```diff
@@ -173,3 +173,12 @@ def setup(self):
 
     def time_masked_column_init(self):
         MaskedColumn(self.dat)
+
+
+class TimeTableInitWithLists:
+
+    def setup(self):
+        self.dat = list(range(100_000))
+
+    def time_init_lists(self):
+        Table([self.dat, self.dat, self.dat], names=['time', 'rate', 'error'])
```

Comment:
NOTE: If `Table` is too smart and knows we're passing the same list that shares memory, I can make copies in setup, pls lemme know.